digits/config/store_option.py

```diff
@@ -29,7 +29,7 @@ def load_url_list():
     if 'DIGITS_MODEL_STORE_URL' in os.environ:
         url_list = os.environ['DIGITS_MODEL_STORE_URL']
     else:
-        url_list = ""
+        url_list = "http://developer.download.nvidia.com/compute/machine-learning/modelstore/5.0"
```

Comment:
I thought you were planning to add the GAN model there. In which case you might want to move to a 6.0 version?